package.json

```diff
@@ -8,7 +8,7 @@
     "email": "info@zoo.dev",
     "url": "https://zoo.dev"
   },
-  "description": "Edit CAD visually or with code",
```

Comment:
Unfortunately this is the quick 'hack' I found because this is what is used by electron-builder and looking at it [I couldn't really find a way to overwrite it](https://github.com/KittyCAD/modeling-app/issues/5050#issuecomment-2598516956). I don't think it matters much though, and this leads to the right result, see https://github.com/KittyCAD/modeling-app/issues/5050#issuecomment-2672435214